docs/style-guide.md

```diff
@@ -1,20 +1,90 @@
 # Style Guide
 
-When contributing to the documentation, please ensure you follow [the AsciiDoc recommended practices].
-They are not currently rigidly enforced.
-However, this may change in the future.
+With regard to English, many things are optional, and style certainly is subjective. This style guide should help all contributors write professionally and consistently. This is a work in progress and will be adjusted as questions arise or suggestions are made.
+
+
+## Basic Writing Rules
+
+- American English is used for all ownCloud documentation, therefore use customize instead of customise, behavior instead of behaviour, etc.
+
+- Don't use 'please'. No need to beg the reader to do something that is necessary to achieve a certain goal.
+
+- Contractions as in "don't" are okay to use.
+
+- Avoid writing "in order to" because "to" says it all.
+
+
+## Use of Lower and Upper Case Letters:
+
+- For headlines, we adhere to the rules of the Chicago Manual of Style. When unsure, use the following tool and select "Do Not Capitalize Words Based on Length":
+https://headlinecapitalization.com/
+
+- When referring to a figure or table in the surrounding text, capitalize Figure 2 or Table 1.
+
+
+## Punctuation:
+
+Some basic punctuation rules in English:
+
+- If a sentence starts with a subordinate clause (like this one does), a comma needs to be set before the main clause starts. Just like in German.
+
+- A main clause (like this one) is followed by a subordinate clause without a comma although in German a comma would be set before the 'although' in this sentence. Simple examples:
+
+When yada happens, you need to use a comma.
+
+Use no comma if dada happens.
+
+- We don't use serial (Oxford) commas in a series of three and more. They are fancy but serve no purpose. Here are examples of serial commas we do NOT want to use, thus placed in []:
+
+Click on the icon, select the desired item from the drop-down menu[,] and click on it.
+
+The icons, the menu[,] and the tool bar are three navigation elements, but don't need a comma before "and".
+
+- As a rule of thumb: you don't need commas before "and", except in rare cases when two main clauses are connected by "and": The banana icon is yellow, and the cucumber icon is green. Other coordinating conjunctions are 'but, or, so, yet'.
+
+If you wonder whether commas are really that important, they can actually change the meaning of a sentence if only in rare cases. Look at the following example with and without the second comma:
+
+A panda enters a bar, eats[,] shoots and leaves.
+
+The comma decides whether the panda eats bamboo shoots or shoots  firearms.
+
+
+## Hyphenated Words
+
+In English, composite terms are rarely hyphenated except for adjectives.
+Some nouns have become one word overtime:
```

Comment:
```suggestion
Some nouns have become one word over time:
```
"overtime" is hours worked in excess of normal working hours :)
IMO this usage should be "over time"

Reply:
Great catch and the perfect context for such an error.